service/src/test/java/uk/nhs/adaptors/gp2gp/TemporaryTest.java

```diff
@@ -0,0 +1,11 @@
+package uk.nhs.adaptors.gp2gp;
+
+import org.junit.jupiter.api.Test;
+
+class TemporaryTest {
```

Comment:
Please remove this class. It does not do anything.

Reply:
It is needed to produce a test report in Jenkins

Reply:
Ok